efficientdet/keras/efficientdet_keras.py

```diff
@@ -569,6 +567,70 @@ def call(self, inputs, training, **kwargs):
     return box_outputs
 
 
+class SegmentationNet(tf.keras.layers.Layer):
```

Comment:
This looks like a head instead of a segmentation network? How about something like "SegmentationHead" or "SegHead"?